src/Form/atoms/ButtonWithLoading.tsx

```diff
@@ -3,20 +3,21 @@ import styled, { css } from 'styled-components';
 
 import Button from './Button';
 import Spinner from '../../Indicators/Spinner';
-import { TypeButtonDesigns, IButtonProps } from '..';
+import { TypeButtonDesigns, TypeButtonSizes, IButtonProps } from '..';
 
 
-const LoadingButton = styled(Button)<{ loading: boolean }>`
-  ${({loading, theme, design}) => loading && css`
+const LoadingButton = styled(Button)<{ loading: boolean, size: TypeButtonSizes }>`
+  ${({loading, theme, size}) => loading && css`
     cursor: wait;
     background: ${ theme.styles.form.button['primary'].active };
-
+    ${(size === 'extraSmall') && css`
+        padding: 3px 20px 3px 10px;
```

Comment:
This is kind of a strange instance I guess so this doesn't need to be perfect, I think we can refactor later.

If you can switch out the `3px` for `0` though that would be great.